tests/providers/amazon/aws/operators/test_batch.py

```diff
@@ -187,6 +188,47 @@ def test_kill_job(self):
         self.client_mock.terminate_job.assert_called_once_with(jobId=JOB_ID, reason="Task killed by the user")
 
 
+class TestBatchOperatorTrimmedArgs:
+    """test class that does not inherit from unittest.TestCase"""
```

Comment:
I added this class before @Taragolis removed `unittest.TestCase` in all AWS tests, this can be merged with the class above now :)